src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -12,32 +13,18 @@ namespace Raven.Server.ServerWide
 {
     public static class ClusterCommandsVersionManager
     {
-        public const int DefaultVersion = 400;
-
         public static readonly int MyCommandsVersion;
 
-        public static int Max(IEnumerable<int> values)
-        {
-            var max = 0;
-            foreach (var value in values)
-            {
-                if (value > max)
-                    max = value;
-            }
-
-            return max;
-        }
-
-        public static int Min(IEnumerable<int> values)
+        public static int Median(int[] values)
```

Comment:
Why use the median here?

Reply:
Cluster version is determined by the highest common version of the majority.
Which is simply the median :-)

Reply:
no, the minimum versions of all the nodes that are there